locales/bn/bn.json

```diff
@@ -143,127 +143,127 @@
     "Congo, the Democratic Republic of the": "কঙ্গো",
     "Constant": "ধ্রুবক",
     "Cook Islands": "কুক দ্বীপপুঞ্জ",
-    "Costa Rica": "ভূটান",
-    "Cote D'Ivoire": "Côte d'Ivoire",
+    "Costa Rica": "কোস্টারিকা",
+    "Cote D'Ivoire": "আইভরি কোট",
     "could not be found.": "খুঁজে পাওয়া যায়নি.",
-    "Country": "Country",
-    "Coupon": "Coupon",
-    "Create": "তৈরি করো",
-    "Create & Add Another": "তৈরি করুন অন্য যোগ করুন",
+    "Country": "দেশ",
+    "Coupon": "কুপন",
+    "Create": "তৈরি করুন",
+    "Create & Add Another": "তৈরি করুন এবং সাথে আরও একটি যোগ করুন",
     "Create :resource": "তৈরি করুন :resource",
     "Create a new team to collaborate with others on projects.": "প্রকল্পে অন্যদের সঙ্গে সহযোগিতা করার জন্য একটি নতুন দল তৈরি করুন.",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`), but here - a period (`.`).

This translation was by machine.